modules/ding_facetbrowser/ding_facetbrowser.module

```diff
@@ -90,23 +90,170 @@ function ding_facetbrowser_ctools_plugin_api($module, $api) {
 }
 
 /**
- * Create a custom form with checkbox elements for facets and ajax callback
+ * Get the interval element.
  *
- * @param array $form
- * @param array $form_state
- * @param array $facets
- * @return array $form
+ * @param \Ting\Search\TingSearchFacet $facet
+ *   The facet.
+ * @param \Ting\Search\TingSearchResultInterface $search_result
+ *   Search result.
+ *
+ * @return array
+ *   An interval form element container.
+ */
+function _ding_facetbrowser_form_interval_element(TingSearchFacet $facet, TingSearchResultInterface $search_result) {
+  // To get the correct counter for the span, we need to run the
+  // search without the actual interval filter on, so the complete dataset
+  // is included.
+  $run_search = FALSE;
+  $range = array();
+
+  // The checksum is used as a cache key for the count request.
+  $checksum = [
```

Comment:
This feels a bit hacky, but it should work for the current search requests.

Another solution was to add a `__sleep` function to the `TingSearchRequest` class. The problem is all other classes connected to the search result (filters, these filters statements and others) should also implement the `__sleep` function, and keep this function updated when changed in the future, this adds extra maintenance for future features.

Reply:
Why do you think you need to implement `__sleep` for `TingSearchRequest`?

I am guessing it is because you wanted to use `serialize` on the request to generate a cache key. That should be fine. Two different objects with the same properties serialize to the same value by default. Using __sleep() also allows you to only serialize a subset of the object's properties. 

Example: https://3v4l.org/oCfC6.